nimble/host/src/ble_gattc.c

```diff
@@ -4413,6 +4413,109 @@ ble_gatts_notify(uint16_t conn_handle, uint16_t chr_val_handle)
     return rc;
 }
 
+int
+ble_gatts_notify_multiple_custom(uint16_t conn_handle,
```

Comment:
if this is exposed to the user, I guess this function should also make sure that remote supports Multiple Notifications,